tests/0.2.0.json

```diff
@@ -0,0 +1,377 @@
+                "$ref": "#/$defs/RecapFloat"
+              },
+              {
+                "$ref": "#/$defs/RecapCommonFields"
+              }
+            ]
+          },
+          {
+            "allOf": [
+              {
+                "$ref": "#/$defs/RecapString"
+              },
+              {
+                "$ref": "#/$defs/RecapCommonFields"
+              }
+            ]
+          },
+          {
+            "allOf": [
+              {
+                "$ref": "#/$defs/RecapBytes"
+              },
+              {
+                "$ref": "#/$defs/RecapCommonFields"
+              }
+            ]
+          },
+          {
+            "allOf": [
+              {
+                "$ref": "#/$defs/RecapList"
+              },
+              {
+                "$ref": "#/$defs/RecapCommonFields"
+              }
+            ]
+          },
+          {
+            "allOf": [
+              {
+                "$ref": "#/$defs/RecapMap"
+              },
+              {
+                "$ref": "#/$defs/RecapCommonFields"
+              }
+            ]
+          },
+          {
+            "allOf": [
+              {
+                "$ref": "#/$defs/RecapStruct"
+              },
+              {
+                "$ref": "#/$defs/RecapCommonFields"
+              }
+            ]
+          },
+          {
+            "allOf": [
+              {
+                "$ref": "#/$defs/RecapEnum"
+              },
+              {
+                "$ref": "#/$defs/RecapCommonFields"
+              }
+            ]
+          },
+          {
+            "allOf": [
+              {
+                "$ref": "#/$defs/RecapUnion"
+              },
+              {
+                "$ref": "#/$defs/RecapCommonFields"
+              }
+            ]
+          }
+        ]
+      },
+      "RecapCommonFields": {
+        "type": "object",
+        "properties": {
+          "alias": {
+            "type": "string"
+          },
+          "doc": {
+            "type": "string"
+          }
+        }
+      },
+      "RecapNull": {
+        "type": "object",
+        "required": [
+          "type"
+        ],
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "null"
+            ]
+          }
+        }
+      },
+      "RecapBool": {
+        "type": "object",
+        "required": [
+          "type"
+        ],
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "bool"
+            ]
+          }
+        }
+      },
+      "RecapInt": {
+        "type": "object",
+        "required": [
+          "type",
+          "bits"
+        ],
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": [
+              "int"
+            ]
+          },
+          "bits": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 64
```

Comment:
I'm not sure about the correct ranges for some of these integer values

Reply:
Integer `bits` field is a 32-bit signed int, so min is 1 and max is 2_147_483_647.

Same for `bits` in float.

`bytes` for string and bytes is 64-bit signed. The `attributes` section in the spec has some info there.

Reply:
updated max for these fields